src/tasks/tasks.service.ts

```diff
@@ -6,22 +6,42 @@ import { TasksRepository } from './tasks.repository';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Task } from './task.entity';
 import { User } from '../auth/user.entity';
+import { PrismaService } from '../prisma.service';
+import { TasksPrismaService } from '../tasks-prisma.service';
+import { task } from '@prisma/client';
+import { user } from '@prisma/client';
 
 @Injectable()
 export class TasksService {
   constructor(
     @InjectRepository(TasksRepository)
     private tasksRepository: TasksRepository,
+    private prismaService: PrismaService,
+    private tasksPrismaService: TasksPrismaService,
+    private prisma: PrismaService,
   ) {}
 
-  getTasks(filterDto: GetTasksFilterDto, user: User): Promise<Task[]> {
-    return this.tasksRepository.getTasks(filterDto, user);
+  getTasks(filterDto: GetTasksFilterDto, user: user): Promise<task[]> {
+    return this.tasksPrismaService.getTasks(filterDto, user);
   }
 
-  async getTaskById(id: string, user: User): Promise<Task> {
+  async getTaskById(id: string, user: user): Promise<task> {
     let found: any;
     // eslint-disable-next-line prefer-const
-    found = await this.tasksRepository.findOne({ where: { id, user } });
+    //found = await this.tasksRepository.findOne({ where: { id, user } });
+    // eslint-disable-next-line prefer-const
+    found = await this.prisma.task.findFirst({
```

Comment:
Would it be possible to use `const` instead of `let`, as it can help to improve the code's readability and maintainability in certain cases?